docs/kr/guides/custom-data-models.md

```
@@ -0,0 +1,134 @@
+    bob = Person(name="Bob")
+    charlie = Person(name="Charlie")
+
+    # Create relationships - field name becomes edge label
+    alice.knows = bob
+    # You can also do lists: alice.knows = [bob, charlie]
+    
+    # Optional: add weights and custom relationship types
+    bob.knows = (Edge(weight=0.9, relationship_type="friend_of"), charlie)
+
+    await add_data_points([alice, bob, charlie])
+
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)